src/daemon/ca/commands.rs

```diff
@@ -63,6 +63,16 @@ pub enum CmdDet {
     // Remove child (also revokes, and removes issued certs, and republishes)
     ChildRemove(ChildHandle),
 
+    // Suspend a child (done by a background process which checks for inactive children)
+    // When a child is inactive it is assumed that they no longer maintain their repository.
+    // The certificate(s) issued to the child will be removed (and revoked) until
+    // the child is seen again and unsuspended (see below).
+    ChildSuspendInactive(ChildHandle),
+
+    // Unsuspend a child (when it contacts the server again). I.e. re-issue
+    // any and all certificates for it based on the current state.
```

Comment:
Unsuspend doesn't re-issue any and all certificates, only those that are not about to expire and do not overclaim, right?